doc/release/0.9.0.md

```diff
@@ -51,7 +51,15 @@ import "./ForgerStakes.sol";
 
 ---
 ## Bug Fixes
-
+- added option to reset modifiers status when comparing forks. This is useful if the node was not updated before the fork
```

Comment:
Maybe it is worth mentioning that this flag should be used just as long as the node has reached the tip of the chain, then the node should be restarted with resetModifiersStatus = false